index.Rmd

```diff
@@ -463,11 +464,11 @@ library(sf)
 source('R/function1.R')
 source('R/function2.R')
 # Load datasets 
-wageningen_ndvi <- rast('data/wag_NDVI.tif')
+wageningen_ndvi <- st_read('data/wag_NDVI.gpkg')
```

Comment:
This is still inconsistent with the text below, you could make it `postbox_locations.gpkg`